core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sentiment/SentimentAnalysisOpDesc.scala

```diff
@@ -1,23 +1,20 @@
 package edu.uci.ics.texera.workflow.operators.sentiment
 
-import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.fasterxml.jackson.annotation.{JacksonAnnotationsInside, JsonProperty, JsonPropertyDescription}
 import com.google.common.base.Preconditions
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaString}
 import edu.uci.ics.amber.engine.architecture.deploysemantics.layer.OpExecConfig
 import edu.uci.ics.amber.engine.common.Constants
-import edu.uci.ics.texera.workflow.common.metadata.{
-  InputPort,
-  OperatorGroupConstants,
-  OperatorInfo,
-  OutputPort
-}
-import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{InputPort, OperatorGroupConstants, OperatorInfo, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.{AutofillAttributeName, SentimentAnalysisAttributeTypeSchema}
 import edu.uci.ics.texera.workflow.common.operators.map.MapOpDesc
 import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, OperatorSchemaInfo, Schema}
 
 import java.util.Collections
 import java.util.Collections.singletonList
 import scala.collection.JavaConverters.{asScalaBuffer, mapAsScalaMap}
 
+@SentimentAnalysisAttributeTypeSchema
```

Comment:
use literal inject as this annotation is unique for one desc to use.

Reply:
Removed.